etc/dbus-serialbattery/default_config.ini

```diff
@@ -107,12 +109,12 @@ INVERT_CURRENT_MEASUREMENT = 1
 ; Every 5% SoC
 ; TIME_TO_SOC_POINTS = [100, 95, 90, 85, 75, 50, 25, 20, 10, 0]
 ; No data set to disable
-TIME_TO_SOC_POINTS =
+TIME_TO_SOC_POINTS=100,95,90,85,75,50,25,20,10,0
 ; Specify TimeToSoc value type: [Valid values 1,2,3]
 ; TIME_TO_SOC_VALUE_TYPE = 1      ; Seconds
 ; TIME_TO_SOC_VALUE_TYPE = 2      ; Time string HH:MN:SC
 ; Both Seconds and time str "<seconds> [days, HR:MN:SC]"
-TIME_TO_SOC_VALUE_TYPE = 3
+TIME_TO_SOC_VALUE_TYPE = 1
```

Comment:
MbItemTimeSpan is based on seconds for proper visualisation see
https://github.com/stgnet/victron-gui-qml/blob/main/MbItemTimeSpan.qml

Reply:
Please see https://github.com/Louisvdw/dbus-serialbattery/pull/468 and https://github.com/Louisvdw/dbus-serialbattery/pull/424#issuecomment-1440511018

Reply:
Thanks for the update. I fixed the issue with dbus-systemcalc. It really requires TimeToGo to only contain an integer value or none. Everything else will break the ESS battery service selection.

Reply:
I will re-base my PR on the improvements PR from you. In general the changes are quite similar and extending upon your PR (additional temp sensors)

Reply:
It would be better, if you merge into jkbms_ble branch and not into master. Since there are a lot of open questions for Bluetooth and not everyone need the Bluetooth packages.